frontend/src/metabase/query_builder/components/DatasetEditor/DatasetNotebook/DatasetNotebook.jsx

```diff
@@ -32,7 +32,7 @@ const Handle = forwardRef(function Handle(props, ref) {
       w="100%"
       h="sm"
       pos="absolute"
-      bottom={`-${rem(4)}`}
+      bottom={rem(-4)}
```

Comment:
`rem` function now outputs `calc` expression, so the style looked like this:
![image](https://github.com/user-attachments/assets/a2988d69-3c0b-474c-a2ce-a25a6e2df14b)
`-calc` did not work.